loaders/mongooseLoader.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
```

Comment:
```suggestion
/* eslint-disable no-console */
const mongoose = require("mongoose");
```
나중에는 no-console "off" 옵션을 바꿔줄꺼니까 미리 주석 넣어놔도 괜찮을 것 같아요!